.github/workflows/gluafixer.yml

```
@@ -1,11 +1,10 @@
 name: GLuaFixer
 
 on:
-  workflow_dispatch:
   pull_request:
-    paths:
-      - 'lua/**'
 
 jobs:
-  lint:
-    uses: CFC-Servers/github_action_workflows/.github/workflows/lint_glua.yml@feature/shareable
+  Lint:
+    uses: FPtje/GLuaFixer/.github/workflows/glualint.yml@master
+    with:
+      config: "https://raw.githubusercontent.com/CFC-Servers/cfc_configs/master/gluafixer/glualint.json"

```

```suggestion
      config: "https://cfc.gg/configs/gluafixer/glualint.json"
```

This is what we use on the other repos